src/Chapter1.hs

```diff
@@ -632,7 +646,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let n2 = abs n
+        ones = mod n2 10
+        tens = div (mod n2 100 - ones) 10
```

Comment:
Great!
For `div` the last digit won't matter, so it is okay not to subtract it in here 🙂